frontend/src/lib/components/Annotations/Annotations.tsx

```diff
@@ -2,24 +2,25 @@ import React from 'react'
 import { annotationsLogic } from './annotationsLogic'
 import { useValues, useActions } from 'kea'
 import { AnnotationMarker } from './AnnotationMarker'
-import { AnnotationType, AnnotationScope } from '~/types'
+import { AnnotationType, AnnotationScope, InsightShortId } from '~/types'
 import { dayjs } from 'lib/dayjs'
 
 interface AnnotationsProps {
     dates: string[]
     leftExtent: number
     interval: number
     topExtent: number
-    insightId?: number
+    insightId?: InsightShortId
+    dashboardItemId?: number // only used for annotations
     color: string | null
     graphColor: string
     accessoryColor: string | null
-    currentDateMarker: string
+    currentDateMarker?: string | null
     onClick: () => void
     onClose: () => void
 }
 
-export function Annotations({
```

Comment:
Converted to render function b/c lint was complaining components cannot return arrays of `JSX.Element`.

Reply:
thats a great idea!